src/DataManagement/MonthlyReport/MonthlyReportCalculator.cs

```diff
@@ -62,27 +67,32 @@ private void CalculateBiggestExpensesCategory()
             var enumCount = Enum.GetValues(typeof(Importance)).Length;
             var biggestCategory = (Importance) enumCount;
             var sum = 0m;
-            for (var i = enumCount; i > (int) Importance.Necessary; i--)
+            foreach (var importance in Enum.GetValues(typeof(Importance)).Cast<Importance>())
             {
                 var temp = 0M;
-                var expenses = Expenses.GetBy((Importance)i);
+                var expenses = Expenses.GetBy((Importance)importance);
```

Comment:
You don't have to cast importance as importance because it already is importance :D

Reply:
That's what I get for only quickly pasting one line from reviews :D whoopps